stdlib/2and3/builtins.pyi

```diff
@@ -1346,15 +1347,55 @@ def next(__i: Iterator[_T], default: _VT) -> Union[_T, _VT]: ...
 def oct(__i: Union[int, _SupportsIndex]) -> str: ...
 
 if sys.version_info >= (3, 6):
-    def open(file: Union[str, bytes, int, _PathLike[Any]], mode: str = ..., buffering: int = ..., encoding: Optional[str] = ...,
-             errors: Optional[str] = ..., newline: Optional[str] = ..., closefd: bool = ...,
-             opener: Optional[Callable[[str, int], int]] = ...) -> IO[Any]: ...
+    # Changed in version 3.6: Support added to accept objects implementing os.PathLike.
+    _OpenFile = Union[str, bytes, int, _PathLike[Any]]
 elif sys.version_info >= (3,):
-    def open(file: Union[str, bytes, int], mode: str = ..., buffering: int = ..., encoding: Optional[str] = ...,
-             errors: Optional[str] = ..., newline: Optional[str] = ..., closefd: bool = ...,
-             opener: Optional[Callable[[str, int], int]] = ...) -> IO[Any]: ...
+    _OpenFile = Union[str, bytes, int]
 else:
-    def open(name: Union[unicode, int], mode: unicode = ..., buffering: int = ...) -> BinaryIO: ...
+    _OpenFile = Union[unicode, int]
+
+if sys.version_info >= (3, 3):
```

Comment:
What about reverting to a `>= (3,):` check to make it explicit that this is a primarily a Python 2/3 distinction? I don't think that we need to care about Python 3.3 and earlier any more, and this initially looked to me that this was mostly about 3.2 and such. A `>= (3,)` would better document the intent and would be less confusing, I think.